src/test/java/mate/academy/security/CustomUserDetailsServiceTest.java

```diff
@@ -0,0 +1,57 @@
+package mate.academy.security;
+
+import java.util.Optional;
+import java.util.Set;
+import mate.academy.model.Role;
+import mate.academy.model.User;
+import mate.academy.service.UserService;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.security.core.userdetails.UserDetailsService;
+import org.springframework.security.core.userdetails.UsernameNotFoundException;
+
+class CustomUserDetailsServiceTest {
+    private static final String EMAIL = "bob@i.ua";
+    private static final String PASSWORD = "12345";
+    private User user;
+    private UserService userService;
```

Comment:
userService can be a local variable.
Better to mock method ones in setUp, because now you have repeated code.